eth-contracts/contracts/Governance.sol

```diff
@@ -696,6 +724,28 @@ contract Governance is InitializableV2 {
         );
     }
 
+     /**
+     * @notice Get proposal description by proposalId
+     * @dev This is a separate function because the getProposalById returns too many
+            variables already and by adding more, you get the error
+            `InternalCompilerError: Stack too deep, try using fewer variables`
+     * @param _proposalId - id of proposal
+     */
+    function getProposalDescriptionById(uint256 _proposalId)
+    external view returns (
+        string memory description
```

Comment:
should this be a named return value? @SidSethi @hareeshnagaraj

Reply:
good catch fixed